app/client/stylesheets/_login.scss

```diff
@@ -1,18 +1,24 @@
 #login {
   height: calc(100% + 49px);
   width: 100%;
-  background-image: url('../assets/images/hero.jpg');
+  background-image: url('../assets/images/login_pg_viv.png'), linear-gradient(to right, #2a2a2a, #191919);
```

Comment:
might be a good idea to rename it to something more descriptive and professional, like `login_bg.png`